src/mame/hp/hp9825.cpp

```diff
@@ -163,20 +162,23 @@ class hp9825_state : public driver_device
 	required_device<hp_09825_67907_cpu_device> m_cpu;
 	required_device_array<hp9825_optrom_device , 4> m_rom_drawers;
 
+	// Printer
+	uint8_t m_printer_mem[ PRINT_COLUMNS ];
+	uint8_t m_printer_idx;
+	unsigned m_printer_line;    // 0: printer idle, 1..10: line being printed
+
+	virtual void print_line();
```

Comment:
Please keep member functions and data members together within an access level section – don’t mix member functions and data members (you have member functions before and after data members in the protected section here).